source/libmvvm_model/mvvm/utils/binutils.cpp

```diff
@@ -0,0 +1,112 @@
+#include "binutils.h"
+
+
+std::string file_check(std::string filename)
+{
+    std::ifstream mySource;
+    mySource.open(filename, std::ios_base::binary);
+    mySource.seekg(0, std::ios_base::end);
+    int size = mySource.tellg();
+    mySource.close();
+    int byte_length = (size>BYTE_LENGTH)?(BYTE_LENGTH):(size);
+
+    std::cout<<"file_size: "<<size<<std::endl;
+
+    std::cout<<"Checking length: "<<byte_length<<std::endl;
+
+    //checking first 1024 bytes
+    int buffer[byte_length];
+    std::ifstream fstr(filename, std::ios::in | std::ios::binary);
+
+    for (int i = 0; i < byte_length; i++)
```

Comment:
Beautify the code with clang-format using `.clang-format` settings located in the root folder of this repository. We have to have all brackets consistent accross the whole projec.